packages/project/src/db/tree.ts

```diff
@@ -42,10 +47,11 @@ export const loadById = async (
     return null;
   }
 
-  const root = JSON.parse(tree.root);
+  const root = denormalizeTree(JSON.parse(tree.root));
   Instance.parse(root);
```

Comment:
btw we should use the return value from .parse call everywhere, I didn't know before but it would allow skipping the data that isn't defined by the schema

Reply:
I didn't know it initially